### PR TITLE
Improve examples

### PR DESCRIPTION
**Pull request by jamowei**
Reworked all examples. Please check if something not right.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,4 +5,4 @@
 **/.idea/
 **/gradle/
 **/src/generated/
-/examples/nestedmodel/model/src/main/kotlin/io/fritz2/examples/nestedmodel/GeneratedOptics.kt
```
Comment (jwstegemann) [inline .gitignore:8]: This should be generated in model-subproject...

File: core/src/main/kotlin/io/fritz2/binding/store.kt [hunk 2]
```diff
@@ -76,3 +113,28 @@ open class RootStore<T>(initialData: T, override val id: String = "", bufferSize
 
     override fun <X> sub(lens: Lens<T, X>) = SubStore(this, lens, this, lens)
 }
+
+
+interface LensId<T> {
```
Comment (jwstegemann) [inline core/src/main/kotlin/io/fritz2/binding/store.kt:118]: I am not sure, if the name explains, what it does. Maybe "ModelId"?

File: core/src/main/kotlin/io/fritz2/dom/mount.kt [hunk 3]
```diff
@@ -80,10 +81,19 @@ class AttributeMountPoint(val name: String, upstream: Flow<String>, val target:
...
+        when (value) {
+            is String -> {
+                target?.unsafeCast<HTMLInputElement>()?.value = value
+            }
+            is Double -> {
+                target?.unsafeCast<HTMLInputElement>()?.valueAsNumber = value
+            }
+            is Date -> {
+                target?.unsafeCast<HTMLInputElement>()?.valueAsDate = value
+            }
```
Comment (jwstegemann) [inline core/src/main/kotlin/io/fritz2/dom/mount.kt:95]: missing else?

**Comment by jwstegemann** [comment]
really great work!
it is good looking and demonstrates perfectly, how it works and feels...

**Comment by jwstegemann** [comment]
Maybe add an examples badge on top of readme?

File: core/src/main/kotlin/io/fritz2/binding/store.kt (hunk 2 above)
Comment (jamowei) [reply core/src/main/kotlin/io/fritz2/binding/store.kt:120]: Renamed in ModelId

File: core/src/main/kotlin/io/fritz2/dom/mount.kt (hunk 3 above)
Comment (jamowei) [reply core/src/main/kotlin/io/fritz2/dom/mount.kt:95]: new concept

**Pull request merged**